.github/workflows/scripts/.pinot_quickstart.sh

```diff
@@ -87,6 +87,11 @@ cd "${DIST_BIN_DIR}"
 
 # Test standalone pinot. Configure JAVA_OPTS for smaller memory, and don't use System.exit
 export JAVA_OPTS="-Xms1G -Dlog4j2.configurationFile=conf/log4j2.xml"
+
+if [ "$JAVA_VER" -gt 15 ]; then
+  JAVA_OPTS="${JAVA_OPTS} --add-opens=java.base/java.nio=ALL-UNNAMED --add-opens=java.base/sun.nio.ch=ALL-UNNAMED"
```

Comment:
By adding this here, these required `add-opens` should be added by default in pinot script if the JVM is > 15